### PR TITLE
Pull request for Altair docs

### PR DESCRIPTION
**Pull request by capac**
Altair is a declarative statistical visualization library for Python, based on Vega and Vega-Lite, and the source is available on GitHub. This is a pull request for the latest version v3.2.0 of Altair.

[Altair: Declarative Visualization in Python](https://altair-viz.github.io/index.html)

### CODE REVIEW

File: docsets/statsmodels/docset.json [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
     },
-    "aliases": ["Statsmodels", "statsmodels", "Statsmodels 0.10.1", "statsmodels 0.10.1"],
+    "aliases": ["Statsmodels", "statsmodels", "Statsmodels 0.10.2", "statsmodels 0.10.2"],
 
     "specific_versions": [
         {
             "version": "0.10",
             "archive": "versions/0.10/statsmodels.tgz",
+            "version": "0.10.1",
+            "archive": "versions/0.10.1/statsmodels.tgz"
```
Comment (Kapeli) [inline docsets/statsmodels/docset.json]: `specific_versions` is supposed to be an array of dictionaries. You added the second version as extra keys within the same dictionary.

Comment (Kapeli) [reply docsets/statsmodels/docset.json]: Also, `specific_versions` need to be sorted by newest-first.

**Comment by Kapeli** [comment]
Thank you! 👍

**Comment by hydrosquall** [comment]
@capac you may find it useful to have Vega and Vega-Lite around as references too.

- https://github.com/Kapeli/Dash-User-Contributions/pull/3383 
- https://github.com/Kapeli/Dash-User-Contributions/pull/3398

**Pull request merged**